corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -164,31 +170,35 @@ hqDefine("geospatial/js/geospatial_map", [
             };
 
             self.addCaseMarkersToMap = function (rawCases) {
-                _.each(rawCases, function(element) {
-                    let coordinates = element.coordinates;
+                const caseColors = {
+                    'default': defaultMarkerColor,
+                    'selected': selectedMarkerColor,
+                };
+
+                _.forEach(rawCases, function (element, caseId) {
+                    const coordinates = element.coordinates;
                     if (coordinates && coordinates.lat && coordinates.lng) {
-                        self.addMarker(element, defaultMarkerColor);
+                        const mapItemInstance = self.addMarker(caseId, element, caseColors);
```

Comment:
nit: since we know this is a case, might as well name this `caseItemInstance` or `caseItem` or `caseMapItem` if we are using `mapItem` instead of `item`.
I believe `item` would make sense if these are just KO objects.

Reply:
That is even better, thanks